### DISCUSSION TITLE
请问如何调试全局扩展脚本

### DISCUSSION THREAD

**Discussion by sagasm** [clash-verge-rev/clash-verge-rev#1908] [Q&A]
1. clash verge设置中打开开发者工具
2. clash verge全局扩展脚本中加入"debugger;"语句
保存脚本之后只是提示: 
  exception: main function should return object
 请问全局脚本该怎么调试?

**Comment by kty00** [comment]
好像只能看控制台吧？

**Comment by BigInventor** [comment]
用console.log() 
<img width="993" height="486" alt="iShot_2026-01-28_10 48 04" src="https://github.com/user-attachments/assets/c1d94c00-0254-4e0a-b961-e9e05370a5ab" />

可以在“全局扩展脚本”这个地方看到日志：
<img width="652" height="117" alt="iShot_2026-01-28_10 48 18" src="https://github.com/user-attachments/assets/b9069078-aac1-41f9-a77b-3b925ee5ba1b" />
<img width="448" height="427" alt="iShot_2026-01-28_10 48 24" src="https://github.com/user-attachments/assets/86ee7b63-c1a6-481f-b4dc-117ecd1b7cc5" />
